src/Titles.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from "react";
+
+class Titles extends Component {
```

Comment:
It would be a good idea to make this a functional component, since it only has `render` method anyway.